Apps/FlightSoftware/Watchdog/include/i2c_sensors.h

```diff
@@ -0,0 +1,107 @@
+// i2c_sensors.h
+
+#ifndef __I2C_SENSORS_H__
+#define __I2C_SENSORS_H__
+
+#include <stdint.h>
+#include <assert.h>
+
+#define I2C_SLAVE_ADDR      100 //0b1100100 //i2c address of LTC2944IDD#PBF fuel gauge
```

Comment:
Is this supposed to be `0b1100100`? `0b1100100` = 200 in decimal. I would rather keep magic numbers in the base that the datasheets specify.

Reply:
Does 0b imply it is a byte? If so my mistake - I meant the binary value 1100100 which is 100 in decimal (double checked with [datasheet](https://www.analog.com/media/en/technical-documentation/data-sheets/2944fa.pdf) page 15 just now)

Reply:
`0b` (or `0B`) implies a [binary constant](https://gcc.gnu.org/onlinedocs/gcc/Binary-constants.html).